src/containers/PageBuilder/SectionBuilder/SectionFooter/SectionFooter.module.css

```diff
@@ -0,0 +1,185 @@
+@import '../../../../styles/customMediaQueries.css';
+
+.footer {
+  /* padding: 32px;
+  max-width: calc(var(--contentMaxWidth) + 72px);
+  margin: 0 auto; */
+  max-width: var(--contentMaxWidthPages);
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 32px;
+  margin: 0 auto;
+  padding: 32px;
+}
+
+.footer ul {
+  list-style-type: none;
+  padding-inline-start: 0px;
+}
+
+.linkList {
+  list-style-type: none;
+  padding-left: 12px;
+}
+
+.content {
+  display: grid;
+  justify-content: center;
+  align-content: end;
+  grid-auto-rows: auto;
+  grid-gap: 32px;
+  grid-row-gap: 16px;
+  grid-template-columns: 1fr;
+  grid-template-areas: 'logo' 'details' 'grid' 'links';
+}
+
+.grid {
+  grid-area: grid;
+  display: grid;
+  justify-content: center;
+  align-content: end;
+  grid-auto-rows: auto;
+  grid-gap: 32px;
+}
+
+.gridCol1 {
+  grid-template-columns: 1fr;
+}
+
+.gridCol2,
+.gridCol3,
+.gridCol4 {
+  grid-template-columns: 1fr 1fr;
+}
+
+.detailsInfo {
+  grid-area: details;
+  & span {
```

Comment:
If you can avoid it, don't use global elements. That leads to "specificity hell" in the long run. (In this context, it's not so bad, because you have used CSS Modules as a wrapper - but as a rule of thumb: instead of styling global elements, create classes, whenever you can.)

In this case, you should be able to just create a new class `.slogan` (no need to nest it)
and then pass the class name to `<Field data={slogan} className={css.slogan} />`

The reason why we style elements in PageBuilder is Markdown syntax, which creates plain HTML elements under the hood. (It's not as straightforward to pass custom classes to those elements.)

Reply:
Thanks for the explanation, fixed!